tests/pds/pds_restore_test.go

```diff
@@ -235,7 +236,7 @@ var _ = Describe("{PerformRestoreToDifferentCluster}", func() {
 				stepLog = "Perform restore for the backup jobs."
 				Step(stepLog, func() {
 					log.InfoD(stepLog)
-					dest_ctx, err := GetDestinationClusterConfigPath()
+					dest_ctx, err := GetSourceClusterConfigPath()
```

Comment:
This should be GetDestinationClusterConfigPath(), since its restoring to a different cluster

Reply:
Good Catch! I have fixed this